altair/examples/cumulative_count_chart.py

```diff
@@ -0,0 +1,21 @@
+"""
+Cumulative Count Chart
+----------------------
+This example shows an area chart with cumulative count.
+Adapted from https://vega.github.io/vega-lite/examples/area_cumulative_freq.html
+
+"""
+# category: area charts
+
+import altair as alt
+from vega_datasets import data
+
+source = data.movies.url
+
+alt.Chart(source).transform_window(
+    cumulative_count="count(count)",
```

Comment:
It would be clearer for this to be ``cumulative_count="count()"`` – no need to use any field name as a standin